### ISSUE TITLE
Trigger on Next and Auto Step properties

### ISSUE THREAD

**Issue by dimitry-ishenko** [CasparCG/Client#172]
Question for devs:
I am in the process of refactoring the Inspector widgets, which is phase 2 of my pull request #171.

As part of it I was going to add Auto Step property for every type of item. But it got me thinking: is the Auto Step property applicable to every item?

I've also noticed that most, but not all items have the Trigger on Next property. Some of them, eg Blend Mode, Brightness, Chroma Key, etc -- don't have it, but might benefit from it.

While going through them, I can add it to the widgets that don't have it, unless there is a specific reason they don't have it. Please let me know.

**Comment by jesperstarkar** [comment]
I think it makes sense to have autostep on all widgets.

**Comment by Keksstar** [comment]
+1 mssing that feature for a long time :(

**Comment by dimitry-ishenko** [comment]
@Keksstar feel free to bug @dotarmin and SVT on this. There are 23 pull requests currently rotting in the cue. I've submitted several requests back in 2016 and eventually just closed them after that sat there for a long time.

As far as I am concerned CCG Client is an abandonware at this point.

**Comment by Keksstar** [comment]
Yes, it´s unclear what´s the status of development is. The Client looks dead for ages. There is little work on the server side if I look on current builds. But maybe @dotarmin can give some informations in the Forum what the current status is.
